### DISCUSSION TITLE
[转]阿里巴巴法务负责人俞思瑛访谈：如何成为好法务？

### DISCUSSION THREAD

**Discussion by wangleo61** [kaiyuanshe/ONES#47] [日常讨论]
https://zhuanlan.zhihu.com/p/103482473

01 法务部门的定位取决于自身的实力
Q：从律师到公司法务，您有什么样的感触？

俞思瑛：对于律师和企业法务在角色定位上的差异，我认为可以从很多角度来回答这个问题。

首先，他们对某个行业理解的深浅程度不一样。

律师通常的思维模式是：这起案例的法律关系是什么？是非法还是合法的？律师更多地是站在法律的角度，结合法律实践，从“合法”和“非法”两个维度去考虑问题。

而企业法务更多是站在企业视角，问自己：这样的问题有没有不通过诉讼解决问题的方法？怎样才能成本更低、效率更高地解决问题？两种角色的基本出发点不太一样。

其次，考虑问题是否关注可持续性。

比如问题发生后，接下来会怎样发展？最开始是怎样的？从律师角度来看，他几乎不太可能参与事情的发生，也未必能参与后面的演变，律师多数是服务于当下的阶段。

但是企业法务从问题一出现就可以介入，一直到最后的危机化解，并且企业法务还需要去反思未来怎么避免此类事件的发生。律师更多的是治病，法务要思考的是如何防患于未然。总结起来就是，律师是单点服务，企业法务是连续跟进。

最后，从个体的角度，企业法务考虑问题的维度会更广，也会更全面。

律师多数时候只考虑法律这个因素就可以了。站在企业的角度，法务还需要考虑媒体舆论、经济成本以及管理问题，对综合能力的要求更高。律师因为代理个案丰富，在同类问题上法律经验的积累也更丰富。

而在综合管理方面，在与企业文化、企业架构这些方面的关联上，对企业法务的要求会更高。虽然都身处法律行业，但是定位不同，所处环境不同，两种角色之间差异还是挺大的。

Q：怎么看法务在整个公司的定位？

俞思瑛：企业的部门可以分为成本中心部门和利润中心部门。对企业来说，当然是希望不发生这些成本好，所以成本中心部门没有那么受重视，但是其实我觉得，法务部门的定位取决于自身的站位，把自己的能动性和价值发挥到最大，这一点很重要。

定位取决于能力，而不是空喊口号。企业法务最重要的能力，是解决问题的能力。这个能力包括调查能力、分析能力、处置能力。一个企业的法务部门最终成了边缘部门，往往是因为法务部门忽略了自身的能力建设。

一名法务既了解商业，又了解法律，还有能力处理相应的危机是很难做到的，精力也不够。所以，当法务部门配备了各种能力的人才，这个法务部门就能够具备相当强的解决问题能力，为公司创造独特的价值。

比如阿里巴巴的法务部，到现在为止有几百人了，这几百人里面有做技术和系统开发的，保证了团队的技术能力；有外部律师资源对接，保证了相应的调查能力；还有新闻部门，保证了对行业的影响力。

所以，“法务是企业的边缘部门，只是用来灭火的，核心部门是利润部门”这种传统说法正在改变，法务部门的定位不是一个结果，而是一个需要努力去改变的过程。

Q：您认为法务部门在企业的整体架构中所起到的作用是怎样的呢？

俞思瑛：法务是一个专业的职能部门，与财务、人力资源等部门类似。像这种专业服务部门，我们都是以“小前端，大后台”的方式去发展。

“小前端”能够快速定位用户的问题，“大后台”则综合了计算能力、监测能力、聚焦能力。“小前端”意味着更多能动性，行动要求团队成员各方面能力都具备，根据实际情况能做出灵活的判断和方案的配置，从而调用大后台的各种能力，由大后台提供解决方案。

在以前，我们认为一名法务必须是全能型的，既会写合同，也会处理案件，解决纠纷、打官司的工作可能也是由同一个人来承担。但现在不一样：今天我们法务部的发展需要非常贴近业务，根据业务方向中最需要的部分，我们可以提出方案，组合团队，整合各种后台资源和业务。

法务团队和其他部门在日常工作中的连接和配合主要体现在三方面：

一是流程管理，在企业运行中，一些流程一定会到法务部门，比如说合同审查、产品评审等，法务部门应该在流程中去管理。

二是价值管理，业务部门不是因为法律纠纷的发生才找到法务部门，而是因为法务部门的工作对其他部门的工作有价值。

三是自驱力管理，自我驱动就是“你不找我，但我认为你需要我，所以我要参与进来，我能够为你贡献价值”。

02 互联网平台的法律责任
Q：近年来“阿里打假”是一个非常热的话题，您怎么看淘宝平台应该承担的法律责任？

俞思瑛：对淘宝来说，淘宝的定位是让更多商户可以进驻，相对而言，商品品质需要消费者去判断和选择。

这个市场的游戏规则是什么？是商品的极大丰富，让消费者想买什么都能买到，这是淘宝平台追求的第一个价值。

第二个价值是，当消费者在买每一样东西的时候，都有无数供应商可以供其选择，让消费者有充分的选择权。同时，消费者可以查看商家信用评价、星级评级、客户评价等，平台也会提供很多工具帮消费者判断商户的好坏，这些工具包括客户评价、“问大家”等平台功能等。另一种平台的定位是，平台帮你选好，能保证交易的绝对安全。

这是两种不同的交易模式，两种不同的市场，有的消费者就喜欢去“淘”，因为平台给予了他们极大的选择空间，也提供了很多工具，所以平台并不能保证每样东西都是检查过的，而是由消费者自己挑选。各种模式的电商已经发展得很充分，而消费者也有自己的常识和能力，依靠平台提供的工具做出判断。

如果要求平台绝对不发生问题，那么只能局限在少数人的范围之内。反之，要覆盖的人群范围越广，平台的门槛和达到的底线相对就比较低。在需要权衡多种因素的情况下，只能综合各种情况，选择一个对整个社会最有利的价值。

今天的社会欢迎的应该是百花齐放的平台。个平台都有自己的定位，有的平台倾向于中立，那就定下规矩：保持开放，让更多用户进来；有的平台倾向于保证消费者的绝对利益，那就定下规矩：提高准入门槛，必须是符合条件的人才能进来。只有当平台选择了不同的定位，对消费者来说才是一个更好的市场，才可以有充分和极大的选择空间，在不同的平台上去寻找喜爱和适合的产品。

03 升级知识产权保护制度
Q：现在人们常争议的网络问题之一是知识产权。互联网时代应当怎样去升级知识产权保护制度呢？

俞思瑛：其实，知识产权的本质是激励我们创新。

当内容经济越来越发达的时候，如何快速实现对内容本身的保护，涉及一个矛盾：一方面知识需要分享，另一方面盗版频繁出现。有价值的内容既需要保护也需要转载和传播，这对我们的技术能力和保护方式提出了更高的挑战。

知识产权保护制度确立的初衷是什么？就是将个人创造变成个人权利，你创造的东西只有拥有了所有权，你的主人翁精神才能够更好地发挥。知识产权让你的价值可以通过跟别人交换实现，可以拿到市场上去应用，去产生相应的影响，收获价值。这种保护，会持续带来正向激励。

但是现在知识产权的整个保护形态，也需要我们去反思和升级。

第一，在网络环境下如何快速地确定权属。我们花了大量时间在辨别原创、辨别真伪上，特别是运用技术手段快速实现权利归属的认定，这是我们要解决的技术问题。

第二，维权能不能更快更有效。现在的规定是，一旦发现侵权，需要权利人通知下架、通知删除，对权利人来讲是很漫长的过程，同时产生很高的社会成本，所以我们要去媒介化，让中间环节变少，让维权成本变低，这是对社会整体利益具有极高价值的事情，也是设计知识产权保护制度过程中需要去思考的。

Q：随着互联网技术的发展，新的商业模式、技术的出现会对法律人的未来产生什么影响？

俞思瑛：很多人都在讨论这个问题。首先。我们需要反思一下，法律人为业务、为商业、为市场贡献的价值是什么？如果解决的是信息不对称的问题，这个价值会被逐步取代，信息不对称随着社会发展会变得越来越少，因为大家都可以自主通过网络去查询、获取信息。以解决信息不对称为主业的法律人，他们的路会越走越窄，这会倒逼我们必须去反思，我们能够为市场和客户创造的价值还有什么。此外，技术的发展对法律人是一件好事情，它让我们提供的服务本身更加接近市场，更加能够解决问题和创造价值。作为法律服务的提供者，必须能够理解正在快速变化的市场，这样才能够跟得上产业、行业的发展，脱离市场的法律服务很难发挥相应价值。

04 创业最缺乏的是找到独特的用户价值
Q：当下创新创业是一个被大家热议的话题，很多年轻人都选择了自己创业。您对想从事互联网创业的年轻人有什么样的经验可以分享？

俞思瑛：首先我觉得，创业已经变得越来越容易了，比以前容易很多。原因是：

第一，很多风险投资商可以帮助你解决初期的资金问题；

第二，很多外包服务商能够帮你解决本来需要你去建团队解决的问题，比如技术、行政工作都可以外包，在这种环境下，需要创业者用足够的差异化和个性化想法去定位，找到独特的用户价值。

一方面创业会变得越来越容易，但同时也会变得越来越难。

整个社会环境让创业变得简单了，以后进入创业市场的人也会越来越多。现在，你可能很难找到一个能挣钱的行业，因为大多数行业已经饱和了。

市场竞争已经越来越激烈，要找到一个独特的客户群体和市场价值，没那么容易，创业者要进入的话，要想清楚你独特的价值在哪里、你的核心竞争力在哪里。

以前，勇气本身就是个门槛，因为有勇气创业的人不多，只要你有勇气创业，成功的概率很高，大部分人死在缺乏勇气这一关。但是现在，创业不缺资金，不缺基础能力的支撑，也不缺勇气，缺乏的是找到独特的用户价值。

Q：要怎样才能找到独特的用户价值呢？

俞思瑛：中国的文化传统追求共性、跟风的取向要多，在经济上也追求大规模的工业化经济，追求精细化商业发展是在这些年发展中体现出来的。

经济发展的同时，也带来了整个社会理念的升级，消费观念的升级，慢慢从标准化的培养模式，发展到了对个性的培养。但这需要很长的过程，人才能慢慢地发现自己的个性，发现自己的喜好，通过自己的喜好去吸引一批粉丝。

现在很多APP都是个人软件开发者，他们追求的是独特的市场和用户价值。今天不是只有做实体的才是创业者，做智力型产品，做软件开发，设计一款产品本身也是创业。

除此之外，还有很多内容创业者的品牌价值，是基于人本身的品牌，内容产业其实也是个人品牌。这是一个非常好的现象，因为社会多元化不仅是一种接纳，更是一种认可，规模化的市场不会像以前那么大。

如果只为了挣钱，不想着能为社会带来多少价值，创业会越来越难，因为缺乏对市场的理解和对客户价值的坚持。创业者只有基于自己的内心、自己的喜爱，以及坚信的用户价值，才有可能会成功。因为总有一些跟你一样的人，他们愿意被你、被你所喜欢的那样东西、被你所发现的价值所打动。

Q：法务团队在企业创新创造过程中，扮演什么样的角色？

俞思瑛：法务作为一个规制部门，对于企业创新发展的作用，是在商业运营方面，业务部门最想法务部门告诉他们的是：我该如何去做。

这一点，其实任何专业都是一样的道理，比如说，有人告诉你，这个事你不能干，那个事你不能干，你肯定会跟他讲，那我能干什么？所以法务工作其实在法律专业和用户（业务）需求之间是需要有个转换的。要站在用户（业务）的角度提出用户（业务）能够做得到的建议，我觉得这才是真正的专业。

我认为，一个真正的好法务不是沉浸在自己的专业话语体系里，而是按照业务和用户的需求做到几点：

第一说人话，第二接地气，最后给出一个商业方案。在平时的工作中，我们也会要求法务给业务部门提出的建议，不要只做法律判断、做法律定性，不能只说合法或非法，而是要针对问题和现象给出能实施的、合理并且成本最低的解决方案。

声明：本文版权归文章作者所有，仅代表作者观点，不代表本公众号观点，仅供学习参考之用。如需转载、节选，请联系作者授权。如文中内容、图片、音频等存在第三方的在先知识产权，请联系微信后台我们会及时删除
发布于 2020-01-19 18:34

**Comment by will-ww** [comment]
好材料，学习了~

**Reply by wangleo61** [reply]
谢谢王老师，多视角下的理解，确实是很有收获
